### PR TITLE
Hasura

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

**Review by guilhermedecampo — APPROVED** [review]

**Review by fakenickels — APPROVED** [review]

**Review by fakenickels — APPROVED** [review]

**Review by guilhermedecampo — COMMENTED** [review]
🚀

File: schema/AddOrganizationMemberMutation.re [hunk 1]
```diff
@@ -49,15 +66,97 @@ let payload =
...
+          );
+        switch (result) {
+        | Result.Ok(content) =>
+          Lwt.return(
+            Ok((
+              None,
+              Some(
+                Member.{
+                  id:
+                    switch (content#id) {
```
Comment (fakenickels) [inline schema/AddOrganizationMemberMutation.re]: eita if this is going to happen a lot we can move to Utils

File: schema/AddOrganizationMemberMutation.re [hunk 2]
```diff
@@ -49,15 +66,97 @@ let payload =
...
+      | Some(user) =>
+        /* OCaml object syntax */
+        let queryInput = {
+          pub email = Some(input.email);
+          pub organizationsByorganizationId = None;
+          pub created_at = None;
+          /* Needed because graphql_ppx defines scalar types to Json by default */
+          pub organization_id =
+            Some(
+              organizationId_to_yojson({
```
Comment (fakenickels) [inline schema/AddOrganizationMemberMutation.re]:
```ocaml
Yojson.to_string(`Assoc("organizationId", `String(input.organizationid)))
```

Comment (fakenickels) [reply schema/AddOrganizationMemberMutation.re]: https://ocaml-community.github.io/yojson/yojson/Yojson/index.html#val-to_string

Comment (fakenickels) [reply schema/AddOrganizationMemberMutation.re]:
```ocaml
Yojson.to_string(`Assoc([ ( "organization_id", `String("foo"))]))
```

**Review by fakenickels — APPROVED** [review]

**Pull request merged**